### PR TITLE
Upgrade Hubot

### PR DESCRIPTION
**Pull request by rubenarakelyan**
This PR upgrades Hubot and dependencies to their latest versions and make some minor adjustments.

### CODE REVIEW

**Review by thomasleese — APPROVED** [review]
This repo looks like it could do with being Dependabot-ed!

File: bin/hubot.cmd [hunk 1]
```diff
@@ -1,3 +1,7 @@
 @echo off
 
-npm install && node_modules\.bin\hubot.cmd --name "hubot" %* 
\ No newline at end of file
+call npm install
+SETLOCAL
```
Comment (bevanloon) [inline bin/hubot.cmd:4]: just curious - what does `SETLOCAL` do?

Comment (rubenarakelyan) [reply bin/hubot.cmd:4]: It means that any env vars set for the rest of the file are only local to the process running the file and are not visible outside.

Comment (timblair) [reply bin/hubot.cmd:4]: https://ss64.com/nt/setlocal.html

**Pull request merged**